packages/requisitions/src/RequestRequisition/DetailView/RequestLineEdit/ItemCharts/ConsumptionHistory.tsx

```diff
@@ -0,0 +1,128 @@
+import React from 'react';
+import {
+  Bar,
+  CartesianGrid,
+  Cell,
+  ChartTooltip,
+  ComposedChart,
+  Legend,
+  Line,
+  Typography,
+  XAxis,
+  YAxis,
+} from '@common/components';
+import {
+  Box,
+  useFormatDate,
+  useTheme,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { DraftRequestLine } from '../hooks';
+
+export interface ConsumptionHistoryProps {
+  draftLine: DraftRequestLine | null;
+}
+
+export const ConsumptionHistory: React.FC<ConsumptionHistoryProps> = ({
+  draftLine,
+}) => {
+  const t = useTranslation('replenishment');
+  const theme = useTheme();
+  const d = useFormatDate();
+  const data = draftLine?.chartData?.consumptionHistory.nodes ?? [];
+  const dateFormatter = (date: string) =>
+    d(new Date(date), {
+      val: { month: 'short', day: '2-digit' },
+    });
+  const tooltipFormatter = (
+    value: number,
+    name: string,
+    props: { payload: { date: string } }
+  ) => {
+    switch (name) {
+      case 'consumption':
+        if (props.payload.date === '2021-03-01')
```

Comment:
uh. yeah. temporary!
and not required really, but makes the tooltip better.
when the data is accurate, can just use the date to get this